dask/dataframe/io/parquet/arrow.py

```diff
@@ -243,8 +243,8 @@ def _collect_pyarrow_dataset_frags(
                 for name in partition_names
             ]
 
-        # Append fragements to our "metadata" list
-        if ds_filters or (split_row_groups and not read_from_paths):
+        # Append fragments to our "metadata" list
+        if ds_filters:
```

Comment:
`read_from_paths` is always `True` now.

Reply:
**Correction**: It is not always `True` - but `read_from_paths` will always be `True` when `ds_filters` is `False` (we only consider passing fragment objects when there are filters). So, this check can still be safely removed.